src/popup/components/TransactionLabel.vue

```diff
@@ -190,7 +195,7 @@ export default defineComponent({
         )) {
           return labelWrapper(root.$t('transaction.dexType.removeLiquidity'));
         }
-        return labelWrapper(addComma(root.$t('transaction.dexType.swap')));
+        return { text: root.$t('transaction.dexType.swap'), hasComma: true };
       }
       if (
         (
```

Comment:
https://github.com/aeternity/superhero-wallet/pull/1856/files Here we have missing commas and translations, can you rebase my branch to you?

Reply:
The comma logic was adjusted to a new approach.